akka-docs/src/main/paradox/remoting.md

```diff
@@ -91,13 +91,13 @@ In the next sections the two alternatives are described in detail.
 Scala
 :   ```
 val selection =
-  context.actorSelection("akka.tcp://actorSystemName@10.0.0.1:2552/user/actorName")
+  context.actorSelection("akka.tcp://actorSystemName@127.0.0.1:2552/user/actorName")
 ```
 
 Java
 :   ```
 ActorSelection selection =
-  context.actorSelection("akka.tcp://app@10.0.0.1:2552/user/serviceA/worker");
```

Comment:
I'm a bit on the fence here as well: indeed we show connecting to 127.0.0.1 in the cluster-usage section because that's the quick way to get 2 actor systems talking to each other. On the other hand, it should be really clear that in production those 2 systems are intended to run on separate nodes...

Perhaps we should try to move them out of here by moving them to environment variables? Though that might introduce another source of complexity...

Reply:
A comment in front of the selection would perhaps make it more clear? `// select an actor on another node`?